doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/project-deployment-options-ref.adoc

```diff
@@ -0,0 +1,57 @@
+[id='project-deployment-options-ref_{context}']
```

Comment:
I would recommend combing this section and the execution section that follows it together as they are closely related and have a lot of duplication.